### ISSUE TITLE
11th i915 driver support in SA6400 model

### ISSUE THREAD

**Issue by dlutwang** [fbelavenuto/arpl#605]
FYI: https://blog.jim.plus/blog/post/jim/synology-sa6400-with-i915

Chinese article, we can understand it with google tranlate.

**Comment by Brian099** [comment]
I checked arpl, SA6400 is added, but it seems not complated

**Comment by dlutwang** [comment]
yes, the solution mentioned in the article was modified by the third party, not fbelavenuto.
you should download the img in the article and do the installation.
